examples/style_hungarian.cf

```diff
@@ -0,0 +1,52 @@
+#+begin_src cfengine3
+bundle agent __main__
+{
+  vars:
+      "s_one"   string => "one";
+      "ITwo"    int => "2";
+      "rThree"  real => "3.0";
+
+      "lMyList" slist => { "$(s_one)", "$(ITwo)", "$(rThree)" };
```

Comment:
I think you should not mix the 3 different styles like this. It seems like you are either encouraging something which users should not do, or your example is noisy (this camelcase vs snakecase vs pascalcase is noise when trying to learn what hungarian notation means).

Reply:
Feel free to change it.